serverless/functions/serverless-functions-getting-started.adoc

```diff
@@ -28,7 +28,7 @@ ifdef::openshift-enterprise[]
 == Additional resources
 * xref:../../registry/securing-exposing-registry.adoc#securing-exposing-registry[Exposing a default registry manually]
 endif::[]
-// need to wait til registry docs are added to OSD for this link to work
+// need to wait til registry docs are added to OSD and ROSA for this link to work
```

Comment:
Same question as above, will registry be available for ROSA at all?

Reply:
I checked if the internal registry is exposed by default on ROSA and it actually is exposed:
```
oc get route.route.openshift.io default-route -n openshift-image-registry
NAME            HOST/PORT                                                                              PATH   SERVICES         PORT    TERMINATION   WILDCARD
default-route   default-route-openshift-image-registry.apps.ocfqe-rosa-102.lh7a.p1.openshiftapps.com          image-registry   <all>   reencrypt     None
```
So it makes no sense to link the `Exposing a default registry manually` document in Additional resources section for ROSA.

Reply:
Seeing the same behaviour on OSD. The registry is there and exposed by default.

Reply:
Perfect, thanks! @rh-max you can actually update this comment then to say that if you want to so that we're aware we won't need to add this link for OSD and ROSA in the future.

Reply:
@abrennan89 Done! Thanks for feedback everyone.